arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -349,6 +347,12 @@ bool VstCommTask::processRead(double startTime) {
   }
 
   if (doExecute) {
+    // A message is complete! Now we need to do the accounting, first let's
+    // set the "read-end time":
+    RequestStatistics* stat = statistics(message.id());
+    RequestStatistics::SET_READ_END(stat);
+    RequestStatistics::ADD_RECEIVED_BYTES(stat, message.totalSize());
```

Comment:
unrelated to this specific line of code, but are including the size of headers in this ?

I wonder whether the server under counts the amount of traffic just because we actually just count the request & response bodies

Reply:
That is right, but I think this will not be a big problem in practice.